src/common/catalog/catalog-entity.ts

```diff
@@ -62,6 +68,19 @@ export interface CatalogCategorySpec {
      */
     kind: string;
   };
+
+  /**
+   * These are the columns used for displaying entities when in the catalog.
+   *
+   * If this is not provided then some default columns will be used, similar in
```

Comment:
Can we document what these "some" are? Maybe point to another file where they're specified.

Reply:
I don't really want to because I don't want to lock in what they are. And from an API perspective I think it is fine that the users just have to try it out to find out.